server/channels/store/sqlstore/channel_store.go

```diff
@@ -2695,7 +2695,7 @@ func (s SqlChannelStore) CountPostsAfter(channelId string, timestamp int64, user
 		})
 
 	if userId != "" {
-		query = query.Where(sq.Eq{"UserId": userId})
```

Comment:
Would you mind renaming this parameter to something like `excludeUserId` that makes it clearer that this works opposite to how it did before?